sbi/inference/snpe/snpe_base.py

```diff
@@ -203,9 +203,6 @@ def train(
 
         max_num_epochs = 2 ** 31 - 1 if max_num_epochs is None else max_num_epochs
 
-        # Load data from most recent round.
-        theta, x, _ = self.get_simulations(self._round, exclude_invalid_x, False)
-
```

Comment:
I think we fixed this already. Which lets it look like you are based on an older `main` branch. Have you tried rebasing your local branch on the most recent version of our `main`?